File: the-river-I..py
# puzzle easy // https://www.codingame.com/ide/puzzle/the-river-i-

r1 = int(input())
r2 = int(input())

while r1 != r2:
    if r1<r2:
        r1+=sum(int(_) for _ in str(r1))
    else:
        r2+=sum(int(_) for _ in str(r2))
print(r1)

File: not Python/the-river-I..rb
# puzzle // https://www.codingame.com/ide/puzzle/the-river-i-

r1 = gets.to_i
r2 = gets.to_i

while r1 != r2 do
  if r1 < r2
    r1+= r1.digits.sum
  else
    r2+= r2.digits.sum
  end  
end

puts r1
